utils/localStorage.js

```diff
@@ -0,0 +1,71 @@
+function removeQueryFromLocalStorage(storageName, term) {
+    var queries = JSON.parse(window.localStorage.getItem(storageName));
+    var filtredQueries = queries.filter(function(query) {
+        if (query === term) {
+            return false;
+        }
+        return true;
+    });
+    window.localStorage.setItem(storageName, JSON.stringify(filtredQueries));
+};
+
+function addQueryToLocalStorage(storageName, query) {
+    if (window.localStorage.getItem(storageName) === null) {
+        window.localStorage.setItem(storageName, JSON.stringify([query]));
+    } else {
+        var queries = JSON.parse(window.localStorage.getItem(storageName));
+        if (!queries.includes(query)) {
+            if (queries.length >= 5) {
```

Comment:
What is the reason of this condition? What does the magic number `5` mean? Use constants for values like this
```js
var SOME_MEANINGFUL_NAME = 5;
```

Reply:
This value is the max numbers of suggestions that will stay in the localStorage. 
Changed to a global variable called `MAX_LOCAL_QUERIES`